src/cli/init.go

```diff
@@ -133,6 +161,21 @@ func (c icli) InitCMD() *cli.Command {
 				Action: c.InitServiceAccountCMD,
 				Before: c.baseBeforeFunc,
 			},
+			{
+				Name:   "knative-domain",
+				Usage:  "updates knative service default domain",
```

Comment:
Can we make clear that a positional argument is required here?

PS. I've tried it with `test.example.com` and the config map now has both the domains, is this expected?

```bash
➜  initium-cli git:(chore/knative_domain) ✗ kubectl get cm config-domain -n knative-serving -o yaml
apiVersion: v1
data:
  example.com: ""
  test.example.com: ""
  ```

Reply:
- I have added a condition to look for an argument and if not to error out with a msg.
- The behaviour is what I mentioned yesterday when you suggested that maybe tilt or argocd reverts to example.com. Will investigate..